lmp/jobserv.yml

```diff
@@ -485,13 +485,6 @@ triggers:
               - jetson-agx-xavier-devkit
               - jetson-agx-orin-devkit
               - qemuarm64-secureboot
-              - raspberrypi4-64
```

Comment:
We don't need to remove rpi4 afaik.

Reply:
We can once we migrate fully to rpi5 I think, but still needs discussion.

Reply:
right; i the I get confused with rpi5 we support on the scarthgap